Tests/Packet++Test/main.cpp

```diff
@@ -7561,6 +7561,32 @@ PTF_TEST_CASE(EthDot3LayerCreateEditTest)
 
 
 
+PTF_TEST_CASE(PacketLayerLookupTest)

```

Comment:
I'd suggest to add a few tests:

- Packet that has a few layers of the same type (you can use `Vxlan1.dat`)
- Try to get prev layer of the first layer (result should be `NULL`)
- Try `getLayerOfType<...>(true)` with a layer type that doesn't exist in the packet

Reply:
I will do it today

Reply:
Done